### DISCUSSION TITLE
The cef plugin inside winfrom

### DISCUSSION THREAD

**Discussion by Guo-yupo** [cefsharp/CefSharp#4465] [General]
Using c# language, the cef plugin inside the winfrom platform can't realize the quick login of enterprise WeChat? What is the reason? But the system default browser shows fine

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask support related questions.

> Specifying a CachePath is required for persistence of cookies, saving of passwords, etc, an In-Memory cache is used by default (similar to Incognito)

https://github.com/cefsharp/CefSharp/wiki/General-Usage#need-to-knowlimitations

**Reply by Guo-yupo** [reply]
The default cache used doesn't seem to work. And I don't know where the cache path is for Enterprise WeChat。

**Reply by amaitland** [reply]
Having never used `WeChat` in any form it's impossible for me to say what's going on. `CEF` doesn't support any enterprise `Chrome` policies.
